### ISSUE TITLE
Dimension Mismatch Issue

### ISSUE THREAD

**Issue by 601SunG** [mousecpn/MMD_AAE_PyTorch#1]
Hi,

Thanks for the codes, but I suppose there may have one potential issue:

In main.py line 151, 
        preds = adv(e.detach())
        adv_loss = advLoss(torch.square(preds), all_labels)
it seems preds has dim 300 while all_labels has 600

Best,
Yang
